docs/source/conf.py

```diff
@@ -52,7 +52,9 @@ def get_build_docset():
 # The default replacements for |version| and |release|, also used in various
 # other places throughout the built documents.
 d = {}
-execfile(os.path.join('..', '..', 'chaco', '_version.py'), d)
+exec(compile(open(os.path.join('..', '..', 'chaco', '_version.py')).read(),
```

Comment:
Can you isolate the `os.path.join('..', '..', 'chaco', '_version.py')` into a local variable for re-use?